### PR TITLE
Fixes #7028 - completing api docs for taxonomies

### PR DESCRIPTION
**Pull request by tstrachota**

### CODE REVIEW

File: app/controllers/concerns/api/v2/taxonomies_controller.rb [hunk 1]
```diff
@@ -15,6 +15,16 @@ module Api::V2::TaxonomiesController
...
     param :resource, Hash, :action_aware => true do
       param :name, String, :required => true
+      param :user_ids, Array, N_("User IDs"), :required => false
+      param :smart_proxy_ids, Array, N_("Smart proxy IDs"), :required => false
+      param :compute_resource_ids, Array, N_("Compute resource IDs"), :required => false
+      param :media_ids, Array, N_("Media IDs"), :required => false
+      param :config_template_ids, Array, N_("Config template IDs"), :required => false
+      param :domain_ids, Array, N_("Domain IDs"), :required => false
+      param :realm_ids, Array, N_("Real IDs"), :required => false
+      param :hostgroup_ids, Array, N_("Hostgroup IDs"), :required => false
```
Comment (domcleal) [inline app/controllers/concerns/api/v2/taxonomies_controller.rb]: "Host group"

File: app/controllers/concerns/api/v2/taxonomies_controller.rb [hunk 2]
```diff
@@ -15,6 +15,16 @@ module Api::V2::TaxonomiesController
   def_param_group :resource do
     param :resource, Hash, :action_aware => true do
       param :name, String, :required => true
+      param :user_ids, Array, N_("User IDs"), :required => false
+      param :smart_proxy_ids, Array, N_("Smart proxy IDs"), :required => false
+      param :compute_resource_ids, Array, N_("Compute resource IDs"), :required => false
+      param :media_ids, Array, N_("Media IDs"), :required => false
+      param :config_template_ids, Array, N_("Config template IDs"), :required => false
+      param :domain_ids, Array, N_("Domain IDs"), :required => false
+      param :realm_ids, Array, N_("Real IDs"), :required => false
```
Comment (domcleal) [inline app/controllers/concerns/api/v2/taxonomies_controller.rb]: Realm

File: app/controllers/concerns/api/v2/taxonomies_controller.rb [hunk 3]
```diff
@@ -15,6 +15,16 @@ module Api::V2::TaxonomiesController
   def_param_group :resource do
     param :resource, Hash, :action_aware => true do
       param :name, String, :required => true
+      param :user_ids, Array, N_("User IDs"), :required => false
+      param :smart_proxy_ids, Array, N_("Smart proxy IDs"), :required => false
+      param :compute_resource_ids, Array, N_("Compute resource IDs"), :required => false
+      param :media_ids, Array, N_("Media IDs"), :required => false
+      param :config_template_ids, Array, N_("Config template IDs"), :required => false
```
Comment (domcleal) [inline app/controllers/concerns/api/v2/taxonomies_controller.rb]: Provisioning template

**Comment by domcleal** [comment]
apipie-rails 0.2.5 is out and built.

**Comment by mbacovsky** [comment]
:+1: Looks good. Fixed the issue.

**Comment by domcleal** [comment]
Thanks @tstrachota, merged as ce690b7 for Foreman 1.6.0.

**Pull request closed without merging**